package.json

```diff
@@ -81,8 +83,5 @@
     "ts-node": "^8.1.0",
     "tsd": "^0.13.1",
     "typescript": "^4.1.0"
-  },
-  "tsd": {
```

Comment:
I removed this section in favour of explicitly specifying the directory to run `tsd` in via the `scripts`' `test:types` run script section above. IMO it is a bit clearer - as a new user of `tsd` myself, I did not know that `tsd` would look for a `tsd` section in `package.json` to look for invocation options.